ssr/src/app.rs

```diff
@@ -128,8 +128,8 @@ pub fn App() -> impl IntoView {
                     // auth redirect routes exist outside main context
                     <GoogleAuthRedirectHandlerRoute/>
                     <GoogleAuthRedirectorRoute/>
-                    <Route path="/" view=RootPage/>
                     <Route path="" view=BaseRoute>
+                        <Route path="" view=RootPage/>
```

Comment:
@rupansh is this still required, since now we are extracting principal from cookie